frontend/src/components/buttons/ReplaceButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from "react";
+import { RefreshSvg } from "../icons/RefreshSvg";
+import styles from "./ReplaceButton.module.scss";
+interface Props {
+  label?: string;
+  onCLick?: () => void;
```

Comment:
```suggestion
  onClick?: () => void;
```

Usages need to be changed as well

Reply:
Can you expand on that? Not sure I follow. Do you mean I shouldn't set a default value for this but instead just leave it as `undefined`?

Reply:
I only mean that there was a spelling mistake, onCLick -> onClick